packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -66,27 +66,31 @@ const navContainerStyle = css`
  * @param props.activeProjectId ID for active project, will cause a POST request to cloud to update current active project.
  * @param props.className Applies a className to the root element
  * @param props.loadData Determines whether or not the component will fetch data from cloud
+ * @param props.activePlatform Determines which platform is active
+ * @param props.alertPollingInterval Defines interval for alert polling
  */
 function MongoNav({
   activeProduct,
   activeNav,
   onOrganizationChange,
   onProjectChange,
+  activeOrgId,
+  activeProjectId,
+  activePlatform,
```

Comment:
I think we can just default this to Cloud and that removes the logic in OrgNav.

Reply:
Then it becomes a required prop in UserMenu, is that what we want?

Reply:
Nvm, that was silly you're right.